### ISSUE TITLE
Order of nn.pixelshuffle

### ISSUE THREAD

**Issue by canornot** [FykAikawa/Anchor-based-SR-pytorch#1]
Nice job. But I suggest line 48 shall be:

upsampled_inp = torch.cat([torch.cat([out[:, [i], ...] for _ in range(self.scale**2)], dim=1) for i in range(self.in_channels)], dim=1)

Because for multi_channel input, order of nn.Pixelshuffle input shall be BBBGGGRRR instead of BGRBGRBGR

**Comment by FykAikawa** [comment]
Thank you for your suggestion!
